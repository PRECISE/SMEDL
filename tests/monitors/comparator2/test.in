nextInt,0
nextInt,1
nextInt,2
nextInt,1
nextFloat,0
nextFloat,1
nextFloat,2
nextFloat,1
nextChar,0
nextChar,1
nextChar,2
nextChar,1
nextString,0
nextString,1
nextString,2
nextString,1
nextPointer,0
nextPointer,1
nextPointer,2
nextPointer,1
nextOpaque,0
nextOpaque,1
nextOpaque,2
nextOpaque,1
