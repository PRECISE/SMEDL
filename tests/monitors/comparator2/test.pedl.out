_IntComp_mismatch,0,1
_IntComp_mismatch,1,2
_IntComp_mismatch,0,2
_IntComp_mismatch,2,1
_IntComp_match,1
_FloatComp_mismatch,0.000000,1.000000
_FloatComp_mismatch,1.000000,2.000000
_FloatComp_mismatch,0.000000,2.000000
_FloatComp_mismatch,2.000000,1.000000
_FloatComp_match,1.000000
_CharComp_mismatch,0,1
_CharComp_mismatch,1,2
_CharComp_mismatch,0,2
_CharComp_mismatch,2,1
_CharComp_match,1
_StringComp_mismatch,0,1
_StringComp_mismatch,1,2
_StringComp_mismatch,0,2
_StringComp_mismatch,2,1
_StringComp_match,1
_PointerComp_mismatch,(nil),0x1
_PointerComp_mismatch,0x1,0x2
_PointerComp_mismatch,(nil),0x2
_PointerComp_mismatch,0x2,0x1
_PointerComp_match,0x1
_OpaqueComp_mismatch,0,1
_OpaqueComp_mismatch,1,2
_OpaqueComp_mismatch,0,2
_OpaqueComp_mismatch,2,1
_OpaqueComp_match,1
